_source/logzio_collections/_log-sources/cloudwatch.md

```diff
@@ -511,6 +506,45 @@ To destroy the resources, use the following:
 ```bash
 terraform destroy
 ```
+</div>
+
+</div>
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="test-events">
+
+#### Working with test events
+
+You can generate test events using the Logz.io Lambda test events generator and add these events to your Lambda function. This functionality is currently only available on Linux & macOS.
+
+
+<div class="tasklist">
+
+##### Generate a test event
+
+In your terminal, run the following command:
```

Comment:
Do you think we should explain that this script generates a test event with a timestamp of the moment they ran the script, in UTC?
And that they should copy the output JSON?